frontend/src/components/Pages/RobotPage/LocalizationSection.tsx

```diff
@@ -8,6 +8,39 @@ interface RobotProps {
 }
 
 export function LocalizationSection({ robot }: RobotProps) {
+    const [selectedAssetDeck, setSelectedAssetDeck] = useState<AssetDeck>()
+    const [assetDecks, setAssetDecks] = useState<AssetDeck[]>()
+
+    useEffect(() => {
+        BackendAPICaller.getAssetDecks().then((response: AssetDeck[]) => {
```

Comment:
should we declare the api caller in the function instead of calling static?

Reply:
I think this is outdated.